pages/index.html

```diff
@@ -1,12 +1,46 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Jennifer's Portfolio Page</title>
+    <link href="/styles/style.css" rel="stylesheet">
+  </head>
+
+  <body>
+    <!-- Nav Bar -->
+    <header>
+      <nav class="nav-menu">
+        <div>
+          <a href="/pages/portfolio.html"> Portfolio </a>
+        </div>
+        <div>
+          <a href="/pages/about.html"> About</a>
+        </div>
+      </nav>
+    </header>
+
+    <!-- hero page -->
+    <main class="hero-page-content">
+      <h2>Hi, I'm Jennifer</h2>
+      <p> and I write <code>#code</code> in:</p>
+      <ul>
```

Comment:
I might style this to remove the bullets and left margin. That would logically keep these a lit, but visually center them along with the rest of the content.